native/.idea/codeStyles/Project.xml

```diff
@@ -19,6 +19,8 @@
       <option name="ALIGN_MULTILINE_PARAMETERS" value="false" />
       <option name="ALIGN_MULTILINE_FOR" value="false" />
       <option name="SPACE_BEFORE_METHOD_PARENTHESES" value="true" />
+      <option name="CALL_PARAMETERS_WRAP" value="1" />
+      <option name="METHOD_PARAMETERS_WRAP" value="1" />
```

Comment:
I think the option we want is chop down if to long:
```suggestion
      <option name="CALL_PARAMETERS_WRAP" value="5" />
      <option name="METHOD_PARAMETERS_WRAP" value="5" />
```
Please verify if this does what I think it does and whether you like it.